src/doc/rustc/src/platform-support.md

```diff
@@ -152,6 +152,7 @@ target | std | notes
 `i586-pc-windows-msvc` | * | 32-bit Windows w/o SSE [^x86_32-floats-x87]
 `i586-unknown-linux-gnu` | ✓ | 32-bit Linux w/o SSE (kernel 3.2, glibc 2.17) [^x86_32-floats-x87]
 `i586-unknown-linux-musl` | ✓ | 32-bit Linux w/o SSE, MUSL [^x86_32-floats-x87]
+[`i586-unknown-netbsd`](platform-support/netbsd.md) |  ✓ | 32-bit x86, restricted to Pentium
```

Comment:
Could you please add a footnote for the usual x86-32 float problem? See the other i686/i586 targets. Depending on whether this target has SSE or not, you need `x86_32-floats-return-ABI` or `x86_32-floats-x87`.